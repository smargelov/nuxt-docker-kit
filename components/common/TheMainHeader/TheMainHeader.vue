<template lang="pug">
  header.b-header
    p._test HEADER
</template>

<script>
export default {
  name: "TheMainHeader",
  props: {},
  data() {
    return {};
  },
  computed: {}
};
</script>

<style lang="sass">
@import './TheMainHeader'
</style>
