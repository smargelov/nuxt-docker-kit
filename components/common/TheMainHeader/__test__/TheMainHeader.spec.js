import { mount } from "@vue/test-utils"
import TheMainHeader from "../TheMainHeader"

describe("TheMainHeader", () => {
  const wrapper = mount(TheMainHeader);
  it("render correctly", () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })
})
