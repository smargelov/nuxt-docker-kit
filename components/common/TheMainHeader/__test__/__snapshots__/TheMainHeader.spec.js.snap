// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheMainHeader render correctly 1`] = `
<header
  class="b-header"
>
  <p
    class="_test"
  >
    HEADER
  </p>
</header>
`;
